zmdocument.cls

```diff
@@ -430,39 +447,68 @@ overlay={\node at (frame.north west) {\createBoxIcon{#5}{#6}};}
    \setcounter{@authorsNumber}{\the@listSize}
    \setcounter{@columnNumber}{\the\numexpr(1 + \the@authorsNumber / \the@authorsByColumn)\relax}
    \thispagestyle{empty}
-   \begin{flushright}
-      \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{titlePageAuthorColor}\bsc{##1}}\\}
-      \ifnum\the@authorsNumber < \the\numexpr(1+\the@authorsByColumn)
+   \Large \bfseries \renewcommand*{\do}[1]{\href{\@authorlink/##1/}{\color{titlePageAuthorColor}\bsc{##1}}\\}
+   \ifnum\the@authorsNumber < \the\numexpr(1+\the@authorsByColumn)
+      \expandafter\docsvlist\expandafter{\@author}
+   \else
+      \begin{multicols}{\the@columnNumber}
          \expandafter\docsvlist\expandafter{\@author}
-      \else
-         \begin{multicols}{\the@columnNumber}
-            \expandafter\docsvlist\expandafter{\@author}
-         \end{multicols}
-      \fi
-   \end{flushright}
+      \end{multicols}
+   \fi
 }
 
 \renewcommand{\maketitle}{%
    \ifcsdef{@authorlink}{}{\authorlink{https://zestedesavoir.com/membres/voir}}
    \ifcsdef{@website}{}{\website{https://zestedesavoir.com}}
    \ifcsdef{@editor}{}{\editor{https://zestedesavoir.com}}
-   \ifcsdef{@editorlogo}{}{\editorlogo{zestedesavoir.png}}
+   \ifcsdef{@editorlogo}{}{\editorlogo{\@logoPath/full.svg}}
```

Comment:
"Full.svg" ? (please use an explicit name instead)

Also (and that was also the case before) but that mean that the compilationf ails if there is not "zestedesavoir.png" or "@LogoPath/full.svg". I would rather prefer a white square or something that does not stops the compilation, but indicate that something is missing. 

Also, [I recently discover](https://en.wikibooks.org/wiki/LaTeX/Creating_Packages#Creating_your_own_class) that you can use `\ClassWarning` (and `\ClassError`) to issue warning and error messages, which is even better :)

Reply:
Editor logo is mandotory, isn'it ?

Reply:
That may be, but then I would rather prefer an explicit error ("you forget to define a editor logo" with `\ClassError`) than assuming that this logo is "zestedesavoir.png" and get a "cryptic error" ("zesetdesavoir.png" not found at a line that would be the one of the `\maketitle`, which is not very explicit)